cli/ops/io.rs

```diff
@@ -27,12 +27,14 @@ use std::os::windows::io::FromRawHandle;
 extern crate winapi;
 
 lazy_static! {
-  /// Due to portability issues on Windows handle to stdout is created from raw file descriptor.
-  /// The caveat of that approach is fact that when this handle is dropped underlying
-  /// file descriptor is closed - that is highly not desirable in case of stdout.
-  /// That's why we store this global handle that is then cloned when obtaining stdio
-  /// for process. In turn when resource table is dropped storing reference to that handle,
-  /// the handle itself won't be closed (so Deno.core.print) will still work.
+  /// Due to portability issues on Windows handle to stdout is created from raw
+  /// file descriptor.  The caveat of that approach is fact that when this
+  /// handle is dropped underlying file descriptor is closed - that is highly
+  /// not desirable in case of stdout.  That's why we store this global handle
+  /// that is then cloned when obtaining stdio for process. In turn when
+  /// resource table is dropped storing reference to that handle, the handle
+  /// itself won't be closed (so Deno.core.print) will still work.
+  // TODO(ry) It should be possible to close stdout.
```

Comment:
Poor phrasing on my part - stdout can be closed using `Deno.close(2)` - what I meant is that dropping a clone of handle won't close stdout.

Reply:
I don't think this static ref to stdout (nor stderr) is necessary, but I don't want to try to remove it in this PR.